docs/architecture/rpc.md

```diff
@@ -0,0 +1,26 @@
+---
+title: The RPC protocol
+authors: "Tim McMackin"
+lastUpdated: 26 October 2023
+---
+
+The Tezos RPC (Remote Procedure Call) protocol is a specification for a REST API that clients use to interact with Tezos nodes and nodes use to communicate with each other.
+Tezos nodes act as servers and accept HTTP requests from clients and other nodes via this protocol.
+
+- Clients use it to submit transactions and get information about the state of the blockchain, such as account balances and contract storage
+- Nodes use it to bake blocks and communicate with other nodes
+
+Tezos RPC uses JSON to send and receive data, but it does not adhere to the JSON-RPC specification.
+
+## RPC nodes
+
+Not all Tezos nodes run RPC servers, and not all RPC servers accept requests from any client.
```

Comment:
I think all nodes have an RPC server (so there is no notion of "RPC nodes"), it's just that their RPC interface is not always open to anyone. What about renaming this section in "Public and private nodes", for instance, and:
```suggestion
The RPC interface of Tezos nodes is subject to an access policy, so not all nodes accept all requests from any client.
```

Reply:
> All Tezos nodes run RPC servers, but the RPC interface is subject to an access policy. By default, RPC servers are private and do not accept all requests from every client.